handout/directives/ng_for_directive.md

```diff
@@ -1,6 +1,6 @@
 # NgFor Directive
 
-The `ngFor` directive is a way of repeating a template by using each item of an iterable as that template's context.
+The `ngFor` directive is a way of repeatly adding content inside the ngFor block to the DOM.
```

Comment:
Reverted to original. Now make sense what Chris was alluding to, forgot out the syntax sugar!